extensions/amp-story/1.0/amp-story-request-service.js

```diff
@@ -55,13 +55,30 @@ export class AmpStoryRequestService {
 
   /**
    * Retrieves the publisher share providers.
-   * Has to be called through `loadShareConfig`.
+   * @param  {!Element} shareConfigEl
```

Comment:
Update `shareConfigEl` name to `configEl` since it can now also represent a shopping config